runtime/lua/vim/fs.lua

```diff
@@ -92,12 +106,14 @@ end
 ---                              reached. The directory itself is not searched.
 ---                       - type (string): Find only files ("file") or
 ---                              directories ("directory"). If omitted, both
----                              files and directories that match {name} are
+---                              files and directories that match {names} are
 ---                              included.
 ---                       - limit (number, default 1): Stop the search after
----                               finding this many matches. Use `math.huge` to
+---                               finding {limit} match(es). Use `math.huge` to
 ---                               place no limit on the number of matches.
----@return (table) The paths of all matching files or directories
+---
+---@return (table) The normalized paths (see last function) of all matching files or directories
```

Comment:
What does "see last function" mean? If you want to link to `vim.fs.normalize()`, then just include that.

```suggestion
---@return (table) The normalized paths |vim.fs.normalize()| of all matching files or directories
```